submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/ExtSubmissionSubmitter.kt

```diff
@@ -44,7 +44,7 @@ class ExtSubmissionSubmitter(
 
     fun indexRequest(accNo: String, version: Int) = requestIndexer.indexRequest(accNo, version)
 
-    fun loadRequest(accNo: String, version: Int): ExtSubmission = requestLoader.loadRequest(accNo, version)
+    fun loadRequest(accNo: String, version: Int) = requestLoader.loadRequest(accNo, version)
```

Comment:
notice others methods does not have return type but the pattern I am follow and I think we agree to is only remove return type in private methods.